### ISSUE TITLE
Implementing R in Jupyter

### ISSUE THREAD

**Issue by noursalam92** [conda/conda#3271]
I am not able to use R in Jupyter:

```
Current conda install:

             platform : win-64
        conda version : 4.1.11
    conda-env version : 2.5.2
  conda-build version : 1.21.3
       python version : 3.5.2.final.0
     requests version : 2.10.0
     root environment : C:\Users\Nour Salam\Anaconda3  (writable)
  default environment : C:\Users\Nour Salam\Anaconda3
     envs directories : C:\Users\Nour Salam\Anaconda3\envs
        package cache : C:\Users\Nour Salam\Anaconda3\pkgs
         channel URLs : https://repo.continuum.io/pkgs/free/win-64/
                        https://repo.continuum.io/pkgs/free/noarch/
                        https://repo.continuum.io/pkgs/pro/win-64/
                        https://repo.continuum.io/pkgs/pro/noarch/
                        https://repo.continuum.io/pkgs/msys2/win-64/
                        https://repo.continuum.io/pkgs/msys2/noarch/
          config file : None
         offline mode : False
    is foreign system : False

```
```

```
```
Traceback (most recent call last):
  File "C:\Users\my_name\Anaconda3\Scripts\conda-script.py", line 5, in <modu
le>
    sys.exit(main())
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\cli\main.py", line
 120, in main
    exit_code = args_func(args, p)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\cli\main.py", line
 130, in args_func
    exit_code = args.func(args, p)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\cli\main_create.py
", line 58, in execute
    install(args, parser, 'create')
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\cli\install.py", l
ine 407, in install
    execute_actions(actions, index, verbose=not args.quiet)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\plan.py", line 599
, in execute_actions
    inst.execute_instructions(plan, index, verbose)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\instructions.py",
line 135, in execute_instructions
    cmd(state, arg)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\instructions.py",
line 59, in EXTRACT_CMD
    extract(arg)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\install.py", line
859, in extract
    exp_backoff_fn(os.rename, temp_path, path)
  File "C:\Users\my_name\Anaconda3\lib\site-packages\conda\install.py", line
245, in exp_backoff_fn
    result = fn(*args)
PermissionError: [WinError 5] Access is denied: 'C:\\Users\\my_name\\Anaconda
3\\pkgs\\r-base-3.3.1-0.tmp' -> 'C:\\Users\\my_name\\Anaconda3\\pkgs\\r-base-
3.3.1-0'

```
```

**Comment by HugoTian** [comment]
Similar to issue #3225 , Permission error

**Comment by noursalam92** [comment]
Hi HugoTian,

Thank you so much for your help. I deactivated my antivirus application (Kaspersky) and retried. It worked! R-essentials was now installed using conda. I created a new environment for R.

However, I still can't figure out why the R does not appear in my 'New' tab in Jupyter. I only have Python and I would like to add more languages such as R. (see image attached)

Could you please help?
Thank you!!!

![image](https://cloud.githubusercontent.com/assets/20956726/17604612/3801c820-5fdc-11e6-9fc9-7f8dd3964048.png)

**Comment by mingwandroid** [comment]
You need to `activate` the new environment, from the `Anaconda Prompt`:

```
activate env_name
jupyter notebook
```

**Comment by noursalam92** [comment]
I tried your suggestion. I then restarted my system (Windows 8.1) and tried it again. 
It didn't work in either cases.
Any ideas on the next step?

Thank you!

**Comment by mingwandroid** [comment]
Can you run `R` from the `Anaconda Prompt` after activating your env? Was `jupyter` installed via `conda`?

**Comment by noursalam92** [comment]
Yes, I can run R in the Anaconda Prompt after activating my env by running the command:
`R.exe`

I installed Anaconda, which installed the following for me: Jupyter, Spyder, IPython, and others.

**Comment by mingwandroid** [comment]
Is this now working for you?

**Comment by noursalam92** [comment]
I still cannot use R in the jupyter notebook, but as I said, I can use it in the Anaconda prompt after I activate my environment.

I would really like to find out how to use it in my jupyter notebook, and I still haven't found the solution

**Comment by mingwandroid** [comment]
From that same anaconda prompt, can you paste the results of:

`which jupyter`

.. and ..

`conda list`

.. and try again to run (from that prompt):

`jupyter notebook`?

**Comment by noursalam92** [comment]
`which jupyter` 

`'which' is not recognized as an internal or external command, operable program or batch file.`

**Comment by mingwandroid** [comment]
Sorry, I meant:

`where jupyter`

**Comment by noursalam92** [comment]
Thank you for your suggestion but that didn't work either.

I tried thinking outside of the box and the problem was solved.
I deleted my newly created 'R-env'.
I then reinstalled the R packages into the root environment.
`conda install -c r r-essentials`
then
`jupyter notebook`

It worked!
![image](https://cloud.githubusercontent.com/assets/20956726/17784439/240c5cca-6542-11e6-8ac6-8c050d32d58a.png)
